compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  sglang:
+    image: lmsysorg/sglang:latest
+    container_name: sglang
+    volumes:
+      - ${HOME}/.cache/huggingface:/root/.cache/huggingface
+    restart: always
+    network_mode: host
+    command:
+      --model-path meta-llama/Meta-Llama-3-8B-Instruct
```

Comment:
nit:
```suggestion
      --model-path meta-llama/Meta-Llama-3.1-8B-Instruct
```

Reply:
Well I just copied it from README.md in main branch, I'll take it.